src/google/protobuf/util/default_value_objectwriter.h

```diff
@@ -0,0 +1,218 @@
+#ifndef NET_PROTO2_UTIL_CONVERTER_INTERNAL_DEFAULT_VALUE_OBJECTWRITER_H_
```

Comment:
Revert this file? It should be in src/google/protobuf/util/internal and seems to be duplicated here.